package.json

```diff
@@ -8,13 +8,15 @@
   "main": "dist/preact-i18n.js",
   "scripts": {
     "lint": "eslint src test",
-    "test": "npm run lint && karma start karma.conf.js --single-run",
+    "test": "npm-run-all lint test:unit",
+    "test:watch": "karma start karma.conf.js --no-single-run",
+    "test:unit": "karma start karma.conf.js --single-run",
     "build": "npm-run-all clean -p rollup:* -p minify:* -s docs size",
     "clean": "rimraf dist && mkdirp dist",
     "rollup:cjs": "rollup -c rollup.config.js -m -f cjs -n $npm_package_amdName $npm_package_jsnext_main -o $npm_package_main",
     "rollup:umd": "rollup -c rollup.config.js -m -f umd -n $npm_package_amdName $npm_package_jsnext_main -o $npm_package_umd_main",
-    "minify:cjs": "uglifyjs $npm_package_main --pure-funcs classCallCheck possibleConstructorReturn -c pure_getters -m toplevel -o $npm_package_main -p relative --in-source-map ${npm_package_main}.map --source-map ${npm_package_main}.map",
-    "minify:umd": "uglifyjs $npm_package_umd_main --pure-funcs classCallCheck possibleConstructorReturn -c pure_getters -m -o $npm_package_umd_main -p relative --in-source-map ${npm_package_umd_main}.map --source-map ${npm_package_umd_main}.map",
+    "minify:cjs": "uglifyjs $npm_package_main -c pure_getters  -m toplevel -o $npm_package_main --source-map content=${npm_package_main}.map,url=preact-i18n.js.map,filename=${npm_package_main}.map",
```

Comment:
removed the `pure-funcs` statements as they had no impact on the final output (I'm guessing they were just copied from another project's config).  The rest is functionally equivalent to the 2.0 config as far as I can tell by diffing the 2.0 and 3.0 build output.